tests/src/Kernel/CoolPreviewFormatterTest.php

```diff
@@ -0,0 +1,216 @@
+<?php
+
+/*
+ * Copyright the Collabora Online contributors.
+ *
+ * SPDX-License-Identifier: MPL-2.0
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+declare(strict_types=1);
+
+namespace Drupal\Tests\collabora_online\Kernel;
+
+use Drupal\Core\Cache\Cache;
+use Drupal\Core\Entity\Display\EntityDisplayInterface;
+use Drupal\entity_test\Entity\EntityTest;
+use Drupal\field\Entity\FieldConfig;
+use Drupal\field\Entity\FieldStorageConfig;
+use Drupal\file\Entity\File;
+use Drupal\media\Entity\Media;
+use Symfony\Component\DomCrawler\Crawler;
+
+/**
+ * @coversDefaultClass \Drupal\collabora_online\Plugin\Field\FieldFormatter\CoolPreview
+ */
+class CoolPreviewFormatterTest extends CollaboraKernelTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected static $modules = [
+    'collabora_online',
+    'key',
+    'media',
+    'user',
+    'field',
+    'system',
+    'file',
+    'image',
+    'entity_test',
+  ];
```

Comment:
No need to repeat modules from the parent class.
```suggestion
  protected static $modules = [
    'entity_test',
  ];
```